UnbxdSearch_IO.js

```diff
@@ -512,6 +512,7 @@ var unbxdSearchInit = function (jQuery, Handlebars) {
         setDefaultFilters: function () {},
         fields: [],
         onNoResult: function (obj) { },
+        onEmptyResult: function (obj) {},
```

Comment:
I think `onEmptySearch` would be a more appropriate name, as this is similar to `onNoResult`

Reply:
Done